### DISCUSSION TITLE
Using first th element on definition table breaks title element

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1756] [Collections (form, grid, menu, message, table, ...)] [answered]
Hello, I'm using a definition table in which I'm using anyway the first `th` element (that should be left empty).
But doing this doesn't make `title` element on it to be showed.

Here is an example: https://jsfiddle.net/q0fsu28e/

As you can see the first `th` of the first table doesn't show titles.

Any way to make it showing?
Thank you.

**Comment by ko2in** [comment, marked as answer]
The first cell of definition table has `pointer-events: none`, so the title cannot be displayed because of the ignorance of the hover event from the browser.

You can override the `pointer-events` in [table.less](https://github.com/fomantic/Fomantic-UI/blob/develop/src/definitions/collections/table.less#L328)

Maybe, we should remove from core as I don't see any side effect with the latest browsers. As my guess, some old browsers may display the cursor when the mouse moves over the first cell if `pointer-events: none` is absent to declare.

What do you think @fomantic/maintainers ? Should we remove this line from table?

**Reply by lubber-de** [reply]
@ko2in We can use both by only disabling pointer events if that cell is empty only. Whenever some kind of content is inside, the pointer events should be kept (imagine somebody tries to use a dropdown in that cell, which obviously need pointer events)

See https://jsfiddle.net/lubber/Lf7nqjx0/5/

So just adding the `:empty` selector to the existing css declaration should do it

**Comment by smartm0use** [comment]
Thank you for your reply. It does the trick ;)

**Comment by ko2in** [comment]
@lubber-de Already submitted the PR to change this behavior as your suggestion.

@smartm0use You'll no longer need to override when the PR got approval and released in next version.
